<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureAlarmDialog</class>
 <widget class="QDialog" name="ConfigureAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>758</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="17,20">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Triggers</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item row="12" column="0">
          <widget class="QCheckBox" name="highHumidityWatch">
           <property name="text">
            <string>High Humidity</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/humidity.png</normaloff>:/icons/ui/humidity.png</iconset>
           </property>
          </widget>
         </item>
         <item row="19" column="0" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when a sensor cannot reliably communicate with the base station</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="lowHumidityWatch">
           <property name="text">
            <string>Low Humidity</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/humidity.png</normaloff>:/icons/ui/humidity.png</iconset>
           </property>
          </widget>
         </item>
         <item row="20" column="0">
          <spacer name="verticalSpacer_8">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="1">
          <widget class="QDoubleSpinBox" name="lowHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="value">
            <double>20.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when the temperature is above the set point</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="highTemperatureWatch">
           <property name="text">
            <string>High Temperature</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/temperature.png</normaloff>:/icons/ui/temperature.png</iconset>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="22" column="0" colspan="2">
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Sends an email to this address when any ov the above conditions triggers</string>
           </property>
          </widget>
         </item>
         <item row="14" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="17" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_10">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Enter a unique name for this alarm</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QDoubleSpinBox" name="highHumidity">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> %RH</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="value">
            <double>80.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when the humidity is below the set point</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="name">
           <property name="text">
            <string>Alarm</string>
           </property>
          </widget>
         </item>
         <item row="16" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when a sensor is low on battery</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="lowTemperatureWatch">
           <property name="text">
            <string>Low Temperature</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/temperature.png</normaloff>:/icons/ui/temperature.png</iconset>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when the temperature is below the set point</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="2">
          <widget class="QLabel" name="label_5">
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Triggered when the humidity is above the set point</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="highTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>40.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="15" column="0" colspan="2">
          <widget class="QCheckBox" name="lowBatteryWatch">
           <property name="text">
            <string>Low Battery</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/battery-0.png</normaloff>:/icons/ui/battery-0.png</iconset>
           </property>
          </widget>
         </item>
         <item row="18" column="0" colspan="2">
          <widget class="QCheckBox" name="lowSignalWatch">
           <property name="text">
            <string>Low Signal</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/signal-0.png</normaloff>:/icons/ui/signal-0.png</iconset>
           </property>
          </widget>
         </item>
         <item row="21" column="0" colspan="2">
          <widget class="QCheckBox" name="sendEmailAction">
           <property name="text">
            <string>Send Email</string>
           </property>
           <property name="icon">
            <iconset resource="../res/res.qrc">
             <normaloff>:/icons/ui/email.png</normaloff>:/icons/ui/email.png</iconset>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="lowTemperature">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string> °C</string>
           </property>
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>-50.000000000000000</double>
           </property>
           <property name="maximum">
            <double>200.000000000000000</double>
           </property>
           <property name="value">
            <double>5.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sensorGroup">
        <property name="title">
         <string>Sensors</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTreeView" name="sensorList">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>lowTemperatureWatch</tabstop>
  <tabstop>lowTemperature</tabstop>
  <tabstop>highTemperatureWatch</tabstop>
  <tabstop>highTemperature</tabstop>
  <tabstop>lowHumidityWatch</tabstop>
  <tabstop>lowHumidity</tabstop>
  <tabstop>highHumidityWatch</tabstop>
  <tabstop>highHumidity</tabstop>
  <tabstop>lowBatteryWatch</tabstop>
  <tabstop>lowSignalWatch</tabstop>
  <tabstop>sendEmailAction</tabstop>
  <tabstop>sensorGroup</tabstop>
  <tabstop>sensorList</tabstop>
 </tabstops>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureAlarmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureAlarmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>717</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowTemperatureWatch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lowTemperature</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highTemperatureWatch</sender>
   <signal>toggled(bool)</signal>
   <receiver>highTemperature</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lowHumidityWatch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lowHumidity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>highHumidityWatch</sender>
   <signal>toggled(bool)</signal>
   <receiver>highHumidity</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
